cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -456,6 +456,7 @@ public static KafkaConnectS2I createEmptyKafkaConnectS2ICluster(String clusterCm
                 .withNamespace(clusterCmNamespace)
                 .withLabels(TestUtils.map(Labels.KUBERNETES_DOMAIN + "part-of", "tests",
                         "my-user-label", "cromulent"))
+                .withAnnotations(emptyMap())
```

Comment:
Why is this now necessary?

Reply:
It seems to be `null` by default. Some tests were relying on non null value. See https://github.com/strimzi/strimzi-kafka-operator/blob/f983bb0fbbab7ce444c123d0eb8d14aaea1e14f2/cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectS2IAssemblyOperatorTest.java#L1023